prefix.go

```diff
@@ -484,6 +486,36 @@ func (i *ipamer) newPrefix(cidr, parentCidr string) (*Prefix, error) {
 	return p, nil
 }
 
+func (i *ipamer) Dump() (string, error) {
+	pfxs, err := i.storage.ReadAllPrefixes()
+	if err != nil {
+		return "", err
+	}
+	js, err := pfxs.toJSON()
+	if err != nil {
+		return "", err
+	}
+	return string(js), nil
+}
+
+func (i *ipamer) Load(dump string) error {
+	pfxs, err := fromJSONs([]byte(dump))
+	if err != nil {
+		return err
+	}
```

Comment:
Maybe we can check before loading if any prefixes already exist. This will make wiping existing data by accident much more unlikely.

Reply:
Yes done